starter-code/movies/models/celebrities.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+const celebritiesSchema = {
```

Comment:
This should be 
const celebritiesSchema = new Schema ({
name : String,
occupation : String,
catchPhrase : String,
} )

Notice that you are extracting the Schema constructor from mongoose, but not using it. It would probably still work if your queries are well built, but you are not actually using the mongoose schema, so you won't have access to its advatages or methods later on.

Reply:
See
http://mongoosejs.com/docs/2.7.x/docs/model-definition.html